agent/src/beerocks/slave/son_slave_thread.cpp

```
@@ -1933,6 +1933,12 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
         std::string client_mac = network_utils::mac_to_string(notification_in->params().mac);
         LOG(INFO) << "client disconnected sta_mac=" << client_mac;
 
+        //update the map to remove this client from its vap
+        auto it = m_vaps_clients_map.find(notification_in->params().bssid);
+        if (it != m_vaps_clients_map.end()) {
```

We might want to throw a warning if the vap is not found...